analyses/cell-type-ewings/cnv-annotation.sh

```diff
@@ -229,7 +222,7 @@ for sce in $data_dir/$sample_id/*_processed.rds; do
     # Run InferCNV
     if [ ! -f "$sample_results_dir/infercnv/${library_id}_cnv-obj.rds" ]; then
       echo "running InferCNV..."
-      Rscript $scripts_dir/run-infercnv.R \
+      Rscript $scripts_dir/04-run-infercnv.R \
         --sce_file "$sce" \
         --annotations_file "$annotations_file" \
         --reference_cell_file "$reference_cell_file" \
```

Comment:
I can't comment, but I happened to notice this one does not use the `$threads` variable

Reply:
Updated in https://github.com/AlexsLemonade/OpenScPCA-analysis/pull/581/commits/7566b2ba5eb7c83834540a76c71e412cefc892d6